CHANGELOG.adoc

```diff
@@ -5,13 +5,13 @@ https://spring.io/projects/spring-cloud-gcp:[Spring Cloud GCP] is a set of integ
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/spring-cloud/spring-cloud-gcp/commits/master[commit history] on GitHub.
 
-== 1.2.0.M3 (2019-10-03) (PRE-RELEASE)
```

Comment:
Should we keep 1.2.0.M3 for historical reference? If someone comes looking for what was different between different releases.

Reply:
There were very few people using milestone releases and we weren't consistent in maintaining the changelog for each one. So, I think it's OK to just focus on more major releases, until someone requests otherwise.